### ISSUE TITLE
Reaper Global Shortcuts bug Behavior (renaming things triggers them)

### ISSUE THREAD

**Issue by atencion** [helgoboss/helgobox#938]
Hi! I`m having trouble renaming things since i have some trouble renaming things in Playtime 2 since whenever i press spacebar, for instance, my PLAY arrangement global shortcut gets trigggered. I think there is a reaper option that will override shortcuts when you press keys in an input field, and that wourl be great for PLaytime to have the same option. It happens with most shortcuts i have globally mapped, like Solo (S) or mute (M) so i can`t really rename things in Playtime withought messing things up.

I`ve tried changing to Prefer Native Pop up menues, but this doesnt change the playtime input input box to reaper native one, so it`s the same.

Hope this description helps, if not, just let me know.

THANKS!

**Comment by helgoboss** [comment]
Hi @atencion. So you added some of your own shortcuts in REAPER's action menu with Scope = "Global" and no modifier key (no ctrl/shift/cmd etc.). And now they are triggered whenever you type something into Playtime's text fields. Is that correct?

**Comment by atencion** [comment]
Exactly. I use a lot of those type of modifiers, like solo mute and specially the SPACEBAR as global so i can control the mix and check different things no matter in wich plugin i`m in. 

So when i`m renaming something in any input field in PLaytime, the corresponding Action to that pressed key shortcut it`s triggered. I wonder if is there any way to let reaper know i`m in a input field so reaper would ignore them as it happens in other input fields in other plugins. Thanks!
